cves/CVE-2015-1872.yml

```
@@ -357,11 +409,13 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: 119
+CWE_note: |
+  Improper restriction of operations within the Bounds oof a Memory Buffer that
+  could allow an attacker to execute arbitrary code or cause the system to crash. 
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it. If the
   report mentions a nickname, e.g. "Heartbleed", use that. Or come up with one!
 
   Must be under 30 characters. Optional. Be appropriate.
-nickname:
+nickname: "Foul Ball Crash"
```

```suggestion
nickname: Foul Ball Crash
```
Values don't need to be wrapped in quotes.